auth/downscoped/initialize_CAB.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package downscopedoverview
+
+// [START auth_downscoping_rules]
+
+import "golang.org/x/oauth2/google/downscope"
+
+// constructCAB shows how to initialize a Credential Access Boundary for downscoping tokens.
+// This sample uses on Access Boundary Rule; up to 10 may be included.
+func constructCAB(bucketName string) {
```

Comment:
This function should also take in an object name, it shouldn't be hardcoded. You can use a substring of the object name in the rule (e.g. objectName[0,3]).

Reply:
In fairness, the object name isn't hard-coded here; there is no object name that's used here.  I'm giving a typical object prefix here, but I don't think I'll change it to take in an object name.  When you construct a CAB, you don't need to know which object (or objects) it'll be used to access.  I'll change the comments to make it more clear that I'm using an example prefix, which could be changed as desired.